webview/src/experiments/components/table/styles.module.scss

```diff
@@ -672,3 +714,26 @@ $badge-size: 0.85rem;
 .cellTooltip {
   padding: 2px 6px;
 }
+
+.indicatorCount {
+  &[aria-label='0'] {
```

Comment:
I tried using only this style to not show the counter when the number is 0, because it would save a conditional test in the component and reduce its complexity.

It works on VSCode and Storybook, but for some reason these styles are not applied when running the jest tests.
So I kept the conditionals in the end.